Source/ThirdParty/ANGLE/src/tests/gl_tests/VariableRasterizationRateTestMetal.mm

```diff
@@ -0,0 +1,300 @@
+//
+// Copyright 2021 The ANGLE Project Authors. All rights reserved.
```

Comment:
2023

Reply:
🤦 That's what happens when copying an existing test to start from.